Tensile/TensileCreateLibrary.py

```diff
@@ -987,13 +989,24 @@ def TensileCreateLibrary():
 
   print1("# CodeObjectVersion from TensileCreateLibrary: %s" % arguments["CodeObjectVersion"])
   print1("# CxxCompiler       from TensileCreateLibrary: %s" % CxxCompiler)
+  print1("# Architecture      from TensileCreateLibrary: %s" % arguments["Architecture"])  
 
   if not os.path.exists(logicPath):
     printExit("LogicPath %s doesn't exist" % logicPath)
 
+  # Translate GPU targets to filter filenames in Tensile_LOGIC directory
+  mapArchitecture = {'all':'_','gfx000':'none', 'gfx803':'r9nano',
+        'gfx900':'vega10', 'gfx906':'vega20', 'gfx908':'arcturus'}
```

Comment:
Don't define these in multiple places. If you need them in more than one spot, put it in common and import it into the files you need like all the other definitions.

Reply:
It seems like I should just remove the map as the part of code that collects the config files is only done in TensileCreateLibrary.py- this isn't called in ClientWriter.py. I'll re-add the line that prints the Tensile_ARCHITECTURE value, but leave out the map.